config/billing_rates.yml

```diff
@@ -354,4 +357,4 @@
 - { id: 765221ad-10cb-8178-9d11-f0268a8be95c, resource_type: PostgresVCpu,                resource_family: standard-m8gd,              location: us-east-2,      unit_price: 0.0017399860, billed_by: duration, active_from: 2025-06-29T00:00:00Z }
 - { id: a2d44646-0f12-48d9-82e7-148ef8963322, resource_type: PostgresStandbyVCpu,         resource_family: standard-m8gd,              location: us-west-2,      unit_price: 0.0017399860, billed_by: duration, active_from: 2025-06-29T00:00:00Z }
 - { id: bd3cc3c3-9b7a-8d78-a318-832d37b8ddd9, resource_type: PostgresStandbyVCpu,         resource_family: standard-m8gd,              location: us-east-1,      unit_price: 0.0017399860, billed_by: duration, active_from: 2025-06-29T00:00:00Z }
-- { id: 3444b23d-4044-8978-8cc5-f80e5620286b, resource_type: PostgresStandbyVCpu,         resource_family: standard-m8gd,              location: us-east-2,      unit_price: 0.0017399860, billed_by: duration, active_from: 2025-06-29T00:00:00Z }
```

Comment:
why is this line removed?

Reply:
It seems like it moved the end of the file somehow, possibly due to conflict resolution. Let me fix that.